src/Compilers/CSharp/Portable/Symbols/Source/SourceMethodSymbolWithAttributes.cs

```diff
@@ -1707,5 +1743,18 @@ internal override System.Reflection.MethodImplAttributes ImplementationAttribute
                 return result;
             }
         }
+
+        internal override int? TryGetOverloadResolutionPriority()
+            => GetEarlyDecodedWellKnownAttributeData()?.OverloadResolutionPriority;
+
+#pragma warning disable IDE0055 // Fix formatting
```

Comment:
nit: Could we link to a known issue? Just want to confirm that the IDE team is aware of the problem and agrees that the current formatting analyzer is incorrect.

Reply:
FYI @CyrusNajmabadi This is a formatting issue we run into regularly. Tracked by https://github.com/dotnet/roslyn/issues/46309 (but it was put in backlog). Should we ask for this to be re-triaged?

Reply:
Never mind, it looks like the issue no longer repro'ed...